packages/manager/modules/pci/src/projects/project/additional-ips/failover-ips/failover-ips.html

```diff
@@ -0,0 +1,94 @@
+<div data-ui-view>
+    <oui-datagrid
+        rows="$ctrl.failoverIps"
+        criteria="$ctrl.criteria"
+        on-criteria-change="$ctrl.onListParamChange($criteria)"
+    >
+        <oui-datagrid-topbar>
+            <oui-action-menu
+                data-text="{{:: 'pci_additional_ips_actions' | translate }}"
+            >
+                <oui-action-menu-item
+                    state="pci.projects.project.additional-ips.imports"
```

Comment:
**question:** Does `state` attribute really exist on `<oui-action-menu-item>` component?
Looks like it doesn't exist https://ovh.github.io/ovh-ui-kit/?path=/story/design-system-components-action-menu-introduction--page#component-oui-action-menu-item.

Reply:
Changed to `on-click`